web/src/scss/layout/_left-panel.scss

```diff
@@ -1,3 +1,32 @@
+#mobile-header {
```

Comment:
I'd just keep this one in `scss/layout/_header.scss` if nothing's changed - makes the PR diff more to the point and easier to review :wink: